ophyd/areadetector/filestore_mixins.py

```diff
@@ -294,6 +392,7 @@ def __init__(self, *args, **kwargs):
                                 ('num_capture', 0),
                                 ])
         self._fn = None
+        self._fp = None
```

Comment:
what is the motivation between `_fn` and `_fp`? I see `_fp` looks like the template and `_fn` the read path after the template. just curious!

Reply:
Because mulit-file tiff datasets.

Reply:
thanks!